File: M8/Exemplos/App_eXperience_Java/app/src/main/java/com/elgin/elginexperience/shipay/requests/AuthenticationRequest.java
package com.elgin.elginexperience.shipay.requests;

import com.google.gson.annotations.SerializedName;

public class AuthenticationRequest {
    public AuthenticationRequest(String access_key, String secret_key, String client_id) {
        this.access_key = access_key;
        this.secret_key = secret_key;
        this.client_id = client_id;
    }
    @SerializedName("access_key")
    public String access_key;

    @SerializedName("secret_key")
    public String secret_key;

    @SerializedName("client_id")
    public String client_id;
}

File: Exemplos/App_eXperience_Kotlin/Elgin_AppExperience_Kotlin-font/app/src/main/java/com/example/e1_kotlin_r/shipay/requests/AuthenticationRequest.kt
package com.example.e1_elgin_kotlin.shipay.requests

import com.google.gson.annotations.SerializedName

data class AuthenticationRequest(
    @field:SerializedName("access_key") var access_key: String,
    @field:SerializedName("secret_key") var secret_key: String,
    @field:SerializedName("client_id") var client_id: String
)
